sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/persistence/DatabasePersistenceAndroidTest.java

```diff
@@ -1223,7 +1320,7 @@ public void upgradeFromVersion2to4() throws PersistenceException, JSONException
             /* Get new data. */
```

Comment:
Line 1309: after check priority migration, need to check migration for the new date field too.

Reply:
I see its done in another method we might want to align tests.

Reply:
nvm its because we had a default value for priority that we were checking.